instrumentation/manager.go

```diff
@@ -27,6 +27,11 @@ type instrumentationDetails[processGroup ProcessGroup, configGroup ConfigGroup]
 	inst Instrumentation
 	pg   processGroup
 	cg   configGroup
+
+	// active is used to track if the instrumentation is loaded successfully or not.
+	// we want to track the instrumentation even if it failed to load, to be able to report the error
+	// and clean up the instrumentation resources and the reporter resources once the process exits.
+	active bool
```

Comment:
consider:

```suggestion
	loaded bool
```

to maintain a consistent terminology with existing terms.